apps/spruce/src/pages/waterfall/BuildRow.tsx

```diff
@@ -48,7 +52,16 @@ export const BuildRow: React.FC<{
   return (
     <Row>
       <BuildVariantTitle>
+        <StyledIconButton
+          active={pinned}
```

Comment:
What do you think about rotating the pin to give it a pin affect? 
<img width="153" alt="Screenshot 2024-11-05 at 9 43 59 AM" src="https://github.com/user-attachments/assets/36028df9-c1fc-4992-afff-16f2f59b2b53">